p5js/ImageClassification/ImageClassification_Preload/sketch.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Image classification using MobileNet and p5.js
+This example uses a callback pattern to create the classifier
+=== */
+
+// A variable to hold the Image Classifier
+let classifier;
+
+// A variable to hold the image we want to classify
+let img;
+
+function preload() {
+  // Initialize the Image Classifier method with MobileNet.
+  classifier = ml5.imageClassifier('mobilenet');
```

Comment:
```suggestion
  classifier = ml5.imageClassifier('MobileNet');
```

Small thing but this is the preferred capitalization I believe.